src/applications/vaos/new-appointment/components/ConfirmationPage/ConfirmationRequestInfo.jsx

```diff
@@ -130,25 +130,21 @@ export default function ConfirmationRequestInfo({
             </dl>
           </div>
           <div className="vads-u-flex--1 vads-u-margin-top--2 vads-u-margin-right--1 vaos-u-word-break--break-word">
-            <dl className="vads-u-margin--0">
-              <dt className="vads-u-font-weight--bold">
-                Preferred date and time
-              </dt>
-              <dd>
-                <ul className="usa-unstyled-list">
-                  {data.calendarData?.selectedDates.map(
-                    ({ date, optionTime }) => (
-                      <li key={`${date}-${optionTime}`}>
-                        {moment(date).format('MMMM D, YYYY')}{' '}
-                        {optionTime === 'AM'
-                          ? 'in the morning'
-                          : 'in the afternoon'}
-                      </li>
-                    ),
-                  )}
-                </ul>
-              </dd>
-            </dl>
+            <h4 className="vads-u-font-size--h4">Preferred date and time</h4>
```

Comment:
Same as above, I think this should be an `h3`.

Reply:
Done.